src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -242,7 +269,7 @@ Tests that we can import an HTMLVideoElement into a GPUExternalTexture and use i
 `
   )
   .fn(async t => {
-    const videoUrl = getResourcePath('red-green.webmvp8.webm');
+    const videoUrl = getResourcePath(kVideoExpectations[0].video);
```

Comment:
Any reason this test can only run this case and not the others?

Reply:
I don't believe there's any restriction. I went ahead and added all of them.